src/Twee/HomeRoutine/Good morning.twee

```diff
@@ -601,7 +601,7 @@ Noticing your worried face at her angry outburst, she quickly reassures you.
 		<<if playerCode.isWearingOn(itemTypes.Chastity)>>\
 			<<if $player.alarmProgress eq 0>>\
 			<<set $player.alarmProgress++>>\
-			a butt-plug alarm vibrating in your anus, bringing you to tears as your locked penis desperately strains against the cage. You try to pull the torturous device from your ass, but it starts to screech and wail so loudly that you push it back in, letting out a weak whimper as you gasp and pant. There is no choice but to endure it, so you bite your pillow and try to relax, but the butt-plug keeps you on the edge the whole time.
+			a butt-plug alarm vibrating in your anus, bringing you to tears as your locked penis desperately strains against the cage. You try to pull the torturous device from your ass, but <<if $kink.bondage>>it starts to screech and wail so loudly that you push it back in, letting out a weak whimper as you gasp and pant.<<else>>it turns out to have inflated on its own, pressing hard against your insides. It won't budge.<<endif>> There is no choice but to endure it, so you bite your pillow and try to relax, but the butt-plug keeps you on the edge the whole time.
```

Comment:
Just going to do an edit of the line.  Also, flipped the condition since "being unable to remove it at all" seems more appropriate for bondage kink than not.

Reply:
```suggestion
			a butt-plug alarm vibrating in your anus, bringing you to tears as your locked penis desperately strains against the cage. You try to pull the torturous device from your ass, but <<if $kinkAllow.bondage>>the plug has inflated on its own, pressing hard against your insides.<<else>>it starts to screech and wail so loudly that you push it back in, letting out a weak whimper as you gasp and pant.<<endif>> There is no choice but to endure it, so you bite your pillow and try to relax, but the butt-plug keeps you on the edge the whole time.
```